issuetracker/src/main/java/org/openjdk/skara/issuetracker/jira/JiraProject.java

```diff
@@ -433,6 +434,18 @@ public Issue createIssue(String title, List<String> body, Map<String, JSONValue>
         }
     }
 
+    @Override
+    public Optional<Issue> jepIssue(String jepId) {
```

Comment:
Have you verified that this works?

Reply:
Not, I have not.

When I request the link `https://bugs.openjdk.java.net/rest/api/2/search?jql=project = JDK AND customfield_10701 = 11` locally, it reports `{"errorMessages":["Field 'customfield_10701' does not exist or you do not have permission to view it."],"errors":{}}`.

But when I request the link `https://bugs.openjdk.java.net/rest/api/2/search?jql=project = JDK AND type = JEP`, it returns the correct data.

It seems that my account don't have permission to search the field `customfield_10701`. Could I get your help to verify it?

Reply:
I opened a JEP issue in the browser and then pressed `F12` to get the HTML info which contains the field name.

Reply:
The link `https://bugs.openjdk.java.net/rest/api/2/search?jql=project = JDK AND type = JEP` also returns this field. Such as `"customfield_10701":"419"`.

Reply:
The previous link `https://bugs.openjdk.java.net/rest/api/2/search?jql=project = JDK AND customfield_10701 = 11` has two errors.

1. We should use the display name to query instead of the internal name. So the `customfield_10701` need to be changed to `JEP Number`.
2. We can't query text field by using the operator `=`. We need to use the operator `~`.

So the link should be `https://bugs.openjdk.java.net/rest/api/2/search?jql=project = JDK AND "JEP Number" ~ "11"`

Related documentation:
[1] https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-issue-search/#api-rest-api-2-search-post
[2] https://support.atlassian.com/jira-software-cloud/docs/advanced-search-reference-jql-operators/

Reply:
I tried to query in the webUI and then convert to advanced query. That gives me:
https://bugs.openjdk.java.net/issues/?jql=project%20%3D%20JDK%20AND%20issuetype%20%3D%20JEP%20AND%20%22JEP%20Number%22%20~%20%2211%22

Trying that with the rest api seems to work too:
https://bugs.openjdk.java.net/rest/api/2/search?jql=project=JDK%20AND%20%22JEP%20Number%22~11

Reply:
Fixed just now.

Reply:
@erikj79 we tried the same thing at the same time and get the same result. So the result should be reliable.

Reply:
When I have been introducing new API calls to Github/Gitlab, I introduced manual tests so that I could verify and debug the interaction with the remote services. These tests required specialized setup that we can't put in the OpenJDK Skara repo (credentials and URLs) and the tests are of a nature that we don't want to run automatically all the time. But at least having something that you can run when you need to, helps a lot. See forge/src/test/java/org/openjdk/skara/forge/ManualForgeTests.java for an example. I don't have any example for Jira. Since this is just a read query, this test won't need authentication a least.

Reply:
I added a test case about the jira api just now.